backend/internal/scores/manager.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/google/uuid"
 )
 
+const scoreEngineMaxLifeTime = 30 * time.Minute
```

Comment:
Read from environment variable and use this constant as default. Change default to hours.